src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -145,7 +146,17 @@ class PublicationReleasePage extends Component<Props> {
                   <strong>{data.releaseName}</strong>
                 </dd>
                 <dd>
-                  <Details summary={`See previous ${releaseCount} releases`}>
+                  <Details
```

Comment:
Looks good to me, if you can find out why the build failed?

Reply:
Looks like I just need to remove a duplicate.

`> explore-education-statistics-frontend@0.1.0 tsc /home/vsts/work/1/s/src/explore-education-statistics-frontend
> tsc --noEmit

src/modules/find-statistics/PublicationReleasePage.tsx(13,10): error TS2300: Duplicate identifier 'logEvent'.
src/modules/find-statistics/PublicationReleasePage.tsx(15,10): error TS2300: Duplicate identifier 'logEvent'.`